karma.conf.js

```diff
@@ -1,6 +1,4 @@
 const webpackConfig = require('./webpack.config.js');
```

Comment:
'coverageReporter' is no more present in Karma config. Have we removed any kind of coverage report for tests?

Reply:
Will check this.

Reply:
Included the `coverage` reporter in karma. Since the tests are being run against the bundles the coverage is also generated against bundles which have very less meaning in terms of lines covered.

The coverage will increase once the tests for `display-controls` are moved. `display-controls` tests have dependency over `ui-control` and I will move them along with `ui-control` in the next PR.

Reply:
What is the coverage %?

Reply:
As of now, the coverage is below. I will be moving tests `displaycontrols` and `models` in subsequent PRs, it will slightly increase.

```
Statements   : 36.56% ( 370/1012 )
Branches     : 24.71% ( 258/1044 )
Functions    : 35.71% ( 200/560 )
Lines        : 100% ( 7/7 )
```